lms/static/scripts/frontend_apps/components/URLPicker.tsx

```diff
@@ -62,7 +57,7 @@ export default function URLPicker({
         <Button
           data-testid="submit-button"
           key="submit"
-          onClick={submit}
+          onClick={() => submit(input.current!.value)}
```

Comment:
I wanted to make the `URLPickerForm` pass the input URL as argument to `onSubmit`, however, there's still the case in which this button is clicked.

Since we still need to provide the input ref to be able to focus it from the `ModalDialog`, I used it to pass the value here, but this is not ideal.